src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import clsx from 'clsx'
+import { hideVisually } from 'polished'
+import { CheckOutline } from 'components/icons'
+import { CheckboxGroup } from './checkbox-group'
+
+/**
+ * Checkbox properties
+ */
+export type CheckboxProps = Omit<
+  React.DetailedHTMLProps<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  >,
+  'checked' | 'disabled' | 'className' | 'style'
+> & {
+  /** HTML checked attribute for input element to set input status true or false */
+  checked?: boolean
```

Comment:
`checked` should be either `boolean | 'indeterminate'` and remove `inderminate` prop